### PR TITLE
[libblastrampoline] Separate LBT builds to coincide with Julia minor versions and build freebsd aarch64 for older versions

### PR DESCRIPTION
**Pull request by ViralBShah**
Try to build LBT 5.4 with freebsd aarch64 support as a result.

cc @ararslan @giordano

### CODE REVIEW

**Review by giordano — COMMENTED** [review]
I'm ok with splitting by version, but I'd prefer reducing duplication by having a `common.jl` script which the other packages include.

File: L/libblastrampoline/libblastrampoline@5.11/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+using BinaryBuilderBase: sanitize
+
```
Comment (giordano) [inline L/libblastrampoline/libblastrampoline@5.11/build_tarballs.jl:5]: I think we can remove these lines, as they are already in the common file
```suggestion
```

File: L/libblastrampoline/libblastrampoline@5.4/build_tarballs.jl [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+using BinaryBuilderBase: sanitize
+
```
Comment (giordano) [inline L/libblastrampoline/libblastrampoline@5.4/build_tarballs.jl:5]:
```suggestion
```

**Review by giordano — APPROVED** [review]

File: L/libblastrampoline/common.jl [hunk 3]
```diff
@@ -7,11 +7,18 @@ name = "libblastrampoline"
 version = v"5.11.1"
```
Comment (giordano) [inline L/libblastrampoline/common.jl:7]: Uhm, this is wrong

**Comment by ararslan** [comment]
Thanks Viral!

**Pull request merged**